src/containers/TriggerTemplate/TriggerTemplate.js

```diff
@@ -0,0 +1,372 @@
+    } = prevProps;
+    const {
+      namespace: prevNamespace,
+      triggerTemplateName: prevTriggerTemplateName
+    } = prevMatch.params;
+
+    if (
+      namespace !== prevNamespace ||
+      triggerTemplateName !== prevTriggerTemplateName ||
+      (webSocketConnected && prevWebSocketConnected === false)
+    ) {
+      this.fetchData();
+    }
+  }
+
+  /* istanbul ignore next */
+  handleYAML = () => {
+    this.setState(prevState => ({
+      showYAML: !prevState.showYAML,
+      showDetails: false
+    }));
+  };
+
+  /* istanbul ignore next */
+  handleDetails = () => {
+    this.setState(prevState => ({
+      showDetails: !prevState.showDetails,
+      showYAML: false
+    }));
+  };
+
+  fetchData() {
+    const { match } = this.props;
+    const { namespace, triggerTemplateName } = match.params;
+    this.props.fetchTriggerTemplate({ name: triggerTemplateName, namespace });
+  }
+
+  render() {
+    const {
+      intl,
+      error,
+      match,
+      selectedNamespace,
+      triggerTemplate
+    } = this.props;
+
+    if (error) {
+      return TriggerTemplateContainer.notification({
+        kind: 'error',
+        message: 'Error loading TriggerTemplate'
+      });
+    }
+    if (!triggerTemplate) {
+      return TriggerTemplateContainer.notification({
+        kind: 'info',
+        message: 'TriggerTemplate not available'
+      });
+    }
+
+    const { params, resourcetemplates } = triggerTemplate.spec;
+    const { triggerTemplateName } = match.params;
+    const triggerTemplateDate = triggerTemplate.metadata.creationTimestamp;
+    const triggerTemplateNamespace = triggerTemplate.metadata.namespace;
+    const triggerTemplateLabels = JSON.stringify(
+      triggerTemplate.metadata.labels
+    );
+
+    const headersForParameters = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerTemplate.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'default',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerTemplate.default',
+          defaultMessage: 'Default'
+        })
+      },
+      {
+        key: 'description',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerTemplate.description',
+          defaultMessage: 'Description'
+        })
+      }
+    ];
+
+    const rowsForParameters = params.map(
+      ({ name, default: defaultValue, description }) => ({
+        id: name,
+        name,
+        default: defaultValue,
+        description
+      })
+    );
+
+    return (
+      <>
+        <h1>{triggerTemplateName}</h1>
+
+        <div className="controlsDiv">
+          <Button onClick={() => this.handleDetails()} kind="ghost">
+            {intl.formatMessage({
+              id: 'dashboard.triggerTemplate.details',
+              defaultMessage: 'Details'
+            })}
+          </Button>
+          <Button onClick={() => this.handleYAML()} kind="ghost">
```

Comment:
`onClick={this.handleYAML}`
no need for the additional anonymous arrow function, it's already bound to the correct scope